README.md

```diff
@@ -25,7 +25,7 @@ A custom rendering pipeline for Canvas for improved visual fidelity.
 
 ## How to Install Lumi Lights
 
-1. Put `LumiLights-vX.XX.zip` in resource packs and activate the resource pack(s).
+1. Put `LumiLights-vX.XX.zip` in resource packs and activate the resource pack(s), and make sure it is above the `canvas_standard` pack.
```

Comment:
It's called `canvas/canvas_default` not `canvas_standard`. And I request adding "to get the full experience" at the end.

![image](https://user-images.githubusercontent.com/8444172/140626262-3b3eb6a2-fe1a-4b3c-9f68-a2a7c062f92a.png)